app/models/local_authority.rb

```diff
@@ -13,6 +13,10 @@ class LocalAuthority < ApplicationRecord
   has_many :links
   belongs_to :parent_local_authority, foreign_key: :parent_local_authority_id, class_name: "LocalAuthority"
 
+  def tier
```

Comment:
For the `tiers` method for Service model we added a unit test, but not for this similar method. Was that intentional as this method is not as complex? Might be a useful thing to document in the tests anyway?